src/components/panel/template.njk

```diff
@@ -1,9 +1,10 @@
+{% set titleLevel = params.titleLevel if params.titleLevel else 1 %}
```

Comment:
If the default is 1, this will be a breaking change, maybe we should default it to 2 and add a deprecation notice?

Reply:
In most cases it will (should) be an h1 so i guess a breaking change.

Reply:
filed under Breaking change

Reply:
If we're going to make this a breaking change we should probably hold this for now and batch it up with other breaking changes.

Reply:
We can make do this work and default to 2, then update the examples in the Design System to explicitly set the level as 1.

We'd then deprecate this behaviour and release the default as 1 in a breaking release later on?

Maybe I'm overcomplicating it.